py/qqa/webpages/templates/logtable.html

```diff
@@ -0,0 +1,75 @@
+{% extends "qabase.html" %}
+{% block body %}
+
+<style>
+label {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  text-align: right;
+  width: 400px;
+  line-height: 26px;
+  margin-bottom: 10px;
+}
+
+input {
+  height: 20px;
+  flex: 0 0 200px;
+  margin-left: 10px;
+}
+
+table {
+  border-collapse: collapse;
+}
+
+.container{
+  display: flex;
+}
+.flex-item{
+  flex-grow: 1;
+}
+
+header {
+  font-weight:bold;
+  font-size: 25px;
+}
+</style>
```

Comment:
logtable.html and logfile.html have duplicated <style> blocks.  Could you structure this so that they inherit a common style?  We must be bumping into one of the organizational reasons that CSS are normally separate files and not embedded into the HTML too.  Consider what the right solution is in this case (I'm not sure).